File: 1-iniciante/1040.cpp
#include <iostream>
#include <cstring>
#include <stdio.h>
using namespace std;
// 
int main()
{
    string inp;
    float *n=new float[4], m,m1;
    getline(cin,inp);
    char *split=strtok(&inp[0]," ");
    for(int i=0;i<4;i++){
        n[i]=stof(split);
        split=strtok(NULL, " ");
    }
    m=((n[0]*2)+(n[1]*3)+(n[2]*4)+n[3])/10;
    printf("Média: %.1f\n", m);
    fflush(stdout);
    if(m<7&&m>=5){
        cout<<"Aluno em exame."<<endl;
        cin>>m1;
        printf("Nota do exame: %.1f\n",m1);
        fflush(stdout);
        if((m+m1)/2>=5){
            cout<<"Aluno aprovado."<<endl;
        } else{
            cout<<"Aluno reprovado."<<endl;
        }
        printf("Mèdia final: %.1f\n",(m+m1)/2);
        fflush(stdout);
    } else if(m<5){
        cout<<"Aluno reprovado."<<endl;
    } else{
        cout<<"Aluno aprovado."<<endl;
    }
    return 0;
}

File: 1-iniciante/1040.py
x=input().split(" ")
m=(float(x[0])*2+float(x[1])*3+float(x[2])*4+float(x[3]))/10
print('Media:',f"{m:.1f}")
if m>=7: print('Aluno aprovado.')
elif m>=5 and m<7:
    print('Aluno em exame.')
    m1=float(input())
    print('Nota do exame:', m1)
    if (m+m1)/2>5: print('Aluno aprovado.')
    else: print('Aluno reporavo.')
    print('Media final:',f"{(m+m1)/2:.1f}")
else:
    print('Aluno reprovado.')